### PR TITLE
Remove `plugin_workspace` from workflow data

### PR DESCRIPTION
**Pull request by hjmodi**
# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [x] Python type annotations added to new code
- n/a JSON/YAML configuration changes are updated in the relevant schema
- n/a Changes to metadata also update the documentation for the metadata
- n/a Pull request has a link to an osbs-docs PR for user documentation updates
- n/a New feature can be disabled from a configuration file

### CODE REVIEW

**Review by rcerven — APPROVED** [review]

**Review by chmeliik — COMMENTED** [review]
LGTM, just one typing suggestion

File: atomic_reactor/plugins/post_koji_import.py [hunk 1]
```diff
@@ -682,6 +670,17 @@ def _update_extra(self, extra):
...
             extra['image']['yum_repourls'] = self.workflow.data.all_yum_repourls
 
+        if is_flatpak_build(self.workflow):
+            flatpak_util = FlatpakUtil(workflow_config=self.workflow.conf,
+                                       source_config=self.workflow.source.config,
+                                       composes=resolve_comp_result['composes'])
+            flatpak_compose_info = flatpak_util.get_flatpak_compose_info()
+            if flatpak_compose_info:
+                koji_metadata = flatpak_compose_info.koji_metadata()
+                koji_metadata['flatpak'] = True
```
Comment (tkdchen) [inline atomic_reactor/plugins/post_koji_import.py:680]: Why not set this directly inside the `koji_metadata()`?

**Review by tkdchen — APPROVED** [review]
LGTM. Just a minor comment.

**Comment by tkdchen** [comment]
By the way, how do you think write markdown format in the commit message? From my perspective, the commit message should be plain text without any git hosting service oriented format, especially in the subject.

'``' is also a bash syntax to execute a command in shell. Would there be any potential issue if it is included in commit message?

File: atomic_reactor/plugins/post_koji_import.py (hunk 1 above)
Comment (hjmodi) [reply atomic_reactor/plugins/post_koji_import.py]: good idea :D. Updated

File: atomic_reactor/utils/flatpak_util.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        return {
+            'source_modules': [self.source_spec],
+            'modules': ['-'.join((m.name, m.stream, m.version)) for
+                        m in sorted_modules if m.name != 'platform'],
+        }
+
+
+class FlatpakUtil:
+    def __init__(self, workflow_config: Configuration, source_config: SourceConfig,
+                 composes=Optional[Dict[str, Any]]):
```
[the remark this answers is not in the document]
Comment (hjmodi) [reply atomic_reactor/utils/flatpak_util.py]: updated.

**Comment by hjmodi** [comment]
> By the way, how do you think write markdown format in the commit message? From my perspective, the commit message should be plain text without any git hosting service oriented format, especially in the subject.
> 
> '``' is also a bash syntax to execute a command in shell. Would there be any potential issue if it is included in commit message?

I don't see that it would be a security issue since I'm pretty sure the hosting service would not execute any command from the commit message and following is my thinking for using ` over '. It's good to be able to read easily from the plugin message which class or data we're updating.

**Pull request merged**